src/cfml/system/services/ServerService.cfc

```diff
@@ -307,16 +328,50 @@ component accessors="true" singleton {
 		// The process native name
 		var processName = serverInfo.name is "" ? "CommandBox" : serverInfo.name;
 
+    var javaagent = serverinfo.cfengine == "lucee" ? "" : "-javaagent:""#libdir#/lucee-inst.jar""";
+    if( serverInfo.cfengine != "lucee" && serverInfo.cfengine != "lucee@" & cfengineVersions["lucee"][1]){
```

Comment:
Find a more elegant way to determine if the server being started is the same version the CLI is running on.